app/assets/javascripts/oxalis/view/action-bar/merge_modal_view.js

```diff
@@ -150,90 +162,93 @@ class MergeModalView extends PureComponent {
+                    {taskType.label}
+                  </Select.Option>,
+                )}
+              </Select>
+            </Form.Item>
+            <Form.Item>
+              <Button type="primary" htmlType="submit" size="default">
+                Merge
+              </Button>
+            </Form.Item>
+          </Form>
+
+          <Form layout="inline" onSubmit={this.handleMergeProject}>
+            <Form.Item label="Project">
+              <Select
+                value={this.state.selectedProject}
+                style={{ width: 200 }}
+                onChange={this.handleChangeMergeProject}
+              >
+                {this.state.projects.map(project =>
+                  <Select.Option key={project} value={project}>
+                    {project}
+                  </Select.Option>,
+                )}
+              </Select>
+            </Form.Item>
+            <Form.Item>
+              <Button type="primary" htmlType="submit" size="default">
+                Merge
+              </Button>
+            </Form.Item>
+          </Form>
+
+          <Form layout="inline">
+            <Form.Item label="NML">
+              <Upload
+                name="nmlFile"
+                action={jsRoutes.controllers.AnnotationIOController.upload().url}
+                headers={{ authorization: "authorization-text" }}
+                beforeUpload={this.handleBeforeUploadNML}
+                onChange={this.handleChangeNML}
+                value={this.state.selectedNML}
+                accept=".nml"
+                showUploadList={false}
+              >
+                <Button icon="upload" style={{ width: 200 }}>
+                  Upload NML and merge
+                </Button>
+              </Upload>
+            </Form.Item>
+          </Form>
+
+          <Form layout="inline" onSubmit={this.handleMergeExplorativeAnnotation}>
+            <Form.Item label="Explorative Annotation">
+              <InputComponent
+                value={this.state.selectedExplorativeAnnotation}
+                style={{ width: 200 }}
+                onChange={this.handleChangeMergeExplorativeAnnotation}
+              />
+            </Form.Item>
+            <Form.Item>
+              <Button type="primary" htmlType="submit" size="default">
+                Merge
               </Button>
-            </Upload>
-          </Form.Item>
-        </Form>
-
-        <Form layout="inline" onSubmit={this.handleMergeExplorativeAnnotation}>
-          <Form.Item label="Explorative Annotation">
-            <InputComponent
-              value={this.state.selectedExplorativeAnnotation}
-              style={{ width: 200 }}
-              onChange={this.handleChangeMergeExplorativeAnnotation}
+            </Form.Item>
+          </Form>
+          <hr />
+          <p>The merged tracing will be saved as a new explorative tracing.</p>
+          {/* <p>
+            <Switch
```

Comment:
Do you remember why we were able to merge tracings as read-only? As a preview option (without saving on server)?